.config/flake8

```diff
@@ -7,4 +7,6 @@ docstring-convention=numpy
 rst-roles=class,func,ref
 rst-directives=envvar,exception
 rst-substitutions=version
-statistics=True
\ No newline at end of file
+statistics=True
+per-file-ignores =
+  /*/interfaces/__init__.py:F401
```

Comment:
Would it be better for this to be directly in the file itself rather than at the top level here?

Reply:
I think perhaps in both.  I'll change it on my internal updates now.